doc/classes/CodeEdit.xml

```diff
@@ -237,12 +237,20 @@
 				Returns the full text with char [code]0xFFFF[/code] at the caret location.
 			</description>
 		</method>
-		<method name="get_text_for_symbol_lookup">
+		<method name="get_text_for_symbol_lookup" qualifiers="const">
 			<return type="String" />
 			<description>
 				Returns the full text with char [code]0xFFFF[/code] at the cursor location.
 			</description>
 		</method>
+		<method name="get_text_with_cursor_char" qualifiers="const">
+			<return type="String" />
+			<param index="0" name="line" type="int" />
+			<param index="1" name="column" type="int" />
+			<description>
+				If the location is not over the text, returns the full text with char [code]0xFFFF[/code] at the specified location, else returns a source text.
```

Comment:
This is independent to `get_text_for_symbol_lookup`, can leave it to this:

Returns the full text with char [code]0xFFFF[/code] at the specified location.